### PR TITLE
AC-728 Should disable 'start visit' in offline mode

### PR DESCRIPTION
**Pull request by LuGO0**
## Description of what I changed
Added a suitable message for the offline mode operation of android client upon starting Active Patient Visit in offline mode

## Issue I worked on
JIRA Issue: https://issues.openmrs.org/browse/AC-728

## Checklist: I completed these to help reviewers :)
- [x] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

- [ ] I have **added tests** to cover my changes. (If you refactored
existing code that was well tested you do not have to add tests)

- [x] All new and existing **tests passed**.

- [x] My pull request is **based on the latest changes** of the master branch.

### CODE REVIEW

**Comment by LuGO0** [comment]
@f4ww4z  please review !!

File: openmrs-client/src/main/java/org/openmrs/mobile/activities/patientdashboard/visits/PatientVisitsFragment.java [hunk 1]
```diff
@@ -68,7 +68,11 @@ public boolean onOptionsItemSelected(MenuItem item) {
         int id = item.getItemId();
         switch (id) {
             case R.id.actionStartVisit:
-                ((PatientDashboardVisitsPresenter) mPresenter).syncVisits();
+                if (OpenMRS.getInstance().getSyncState()) {
+                    ((PatientDashboardVisitsPresenter) mPresenter).syncVisits();
+                } else {
+                    ToastUtil.notify("Active patient visit cannot be started in offline mode");
```
Comment (rishabh-997) [inline openmrs-client/src/main/java/org/openmrs/mobile/activities/patientdashboard/visits/PatientVisitsFragment.java:74]: Add this to the `string` resource file

Comment (rishabh-997) [reply openmrs-client/src/main/java/org/openmrs/mobile/activities/patientdashboard/visits/PatientVisitsFragment.java:74]: Also, add JIRA ticket to title

Comment (LuGO0) [reply openmrs-client/src/main/java/org/openmrs/mobile/activities/patientdashboard/visits/PatientVisitsFragment.java:74]: yes @rishabh-997 sure !!

Comment (f4ww4z) [reply openmrs-client/src/main/java/org/openmrs/mobile/activities/patientdashboard/visits/PatientVisitsFragment.java:74]: @LuGO0 this is still not in a string resource file.

Comment (LuGO0) [reply openmrs-client/src/main/java/org/openmrs/mobile/activities/patientdashboard/visits/PatientVisitsFragment.java]: @f4ww4z added it !!

**Comment by f4ww4z** [comment]
@LuGO0 The title of the PR should always be identical to the JIRA issue title, keep this in mind next time.

LGTM.

**Pull request merged**